rswag-specs/lib/rswag/specs.rb

```diff
@@ -12,8 +12,7 @@ module Specs
       swagger_root: :openapi_root,
       swagger_docs: :openapi_specs,
       swagger_dry_run: :rswag_dry_run,
-      swagger_format: :openapi_format,
-      swagger_strict_schema_validation: :openapi_strict_schema_validation
```

Comment:
I'm removing this because it implies that `swagger_strict_schema_validation` is being renamed to `openapi_strict_schema_validation`